test/entice/logic/skillbar_test.exs

```diff
@@ -103,4 +104,23 @@ defmodule Entice.Logic.SkillBarTest do
     assert {:error, :still_recharging} = SkillBar.cast_skill(eid, 0, &(send this, &1), &(send this, &1))
     assert {:ok, :instant, _skill}     = SkillBar.cast_skill(eid, 1, &(send this, &1), &(send this, &1))
   end
+
+  test "cast skill with enough energy", %{entity_id: eid} do
+    assert [6,0,0,0,0,0,0,0] = SkillBar.change_skill(eid, 0, 6) # switch to mantra of earth
+
+    this = self
+
+    assert {:ok, :instant, _skill} = SkillBar.cast_skill(eid, 0, &(send this, &1), &(send this, &1))
```

Comment:
`&(send this, &1)` can be `&(&1)` in both cases, and the above `this = self` can be removed ;)